packages/config/config/devices/0x0438/4512757.json

```diff
@@ -0,0 +1,534 @@
+					"value": 30
+				},
+				{
+					"label": "40%",
+					"value": 40
+				},
+				{
+					"label": "50%",
+					"value": 50
+				},
+				{
+					"label": "60%",
+					"value": 60
+				},
+				{
+					"label": "70%",
+					"value": 70
+				},
+				{
+					"label": "80%",
+					"value": 80
+				},
+				{
+					"label": "90%",
+					"value": 90
+				},
+				{
+					"label": "100%",
+					"value": 100
+				}
+			]
+		},
+		{
+			"#": "124",
+			"label": "LCD Backlight work",
+			"description": "Allowable range: 10-100 (increment by 10)",
+			"valueSize": 4,
+			"defaultValue": 100,
+			"minValue": 10,
+			"maxValue": 100,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "10%",
+					"value": 10
+				},
+				{
+					"label": "20%",
+					"value": 20
+				},
+				{
+					"label": "30%",
+					"value": 30
+				},
+				{
+					"label": "40%",
+					"value": 40
+				},
+				{
+					"label": "50%",
+					"value": 50
+				},
+				{
+					"label": "60%",
+					"value": 60
+				},
+				{
+					"label": "70%",
+					"value": 70
+				},
+				{
+					"label": "80%",
+					"value": 80
+				},
+				{
+					"label": "90%",
+					"value": 90
+				},
+				{
+					"label": "100%",
+					"value": 100
+				}
+			]
+		},
+		{
+			"#": "127",
+			"label": "Regulator percentage",
+			"description": "Allowable range: 0-100 (increment by 10)",
+			"valueSize": 4,
+			"defaultValue": 20,
+			"minValue": 0,
+			"maxValue": 100,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "0%",
+					"value": 0
+				},
+				{
+					"label": "10%",
+					"value": 10
+				},
+				{
+					"label": "20%",
+					"value": 20
+				},
+				{
+					"label": "30%",
+					"value": 30
+				},
+				{
+					"label": "40%",
+					"value": 40
+				},
+				{
+					"label": "50%",
+					"value": 50
+				},
+				{
+					"label": "60%",
+					"value": 60
+				},
+				{
+					"label": "70%",
+					"value": 70
+				},
+				{
+					"label": "80%",
+					"value": 80
+				},
+				{
+					"label": "90%",
+					"value": 90
+				},
+				{
+					"label": "100%",
+					"value": 100
+				}
+			]
+		},
+		{
+			"#": "134",
+			"label": "DIF Celsius",
+			"description": "Allowable range: 5-100 (increment by 5)",
+			"valueSize": 4,
+			"defaultValue": 0,
+			"minValue": 5,
+			"maxValue": 100,
+			"unsigned": true
+		},
+		{
+			"#": "135",
+			"label": "DIF Fahrenheit",
+			"description": "Allowable range: 10-180 (increment by 10)",
+			"valueSize": 4,
+			"defaultValue": 20,
+			"minValue": 10,
+			"maxValue": 180,
+			"unsigned": true
+		},
+		{
+			"#": "136",
+			"label": "Frost Celsius",
+			"description": "Allowable range: 50-100 (increment by 10)",
+			"valueSize": 4,
+			"defaultValue": 50,
+			"minValue": 50,
+			"maxValue": 100,
+			"unsigned": true
+		},
+		{
+			"#": "137",
+			"label": "Frost Fahrenheit",
+			"description": "Allowable range: 410-500 (increment by 20)",
+			"valueSize": 4,
+			"defaultValue": 410,
+			"minValue": 410,
+			"maxValue": 500,
+			"unsigned": true
+		},
```

Comment:
Those values seem high for frost. Could it be that they are in 1/10 degrees? If so, we usually specify that multiplier in the unit:
```
"unit": "0.1 °C",
```

Reply:
You are absolutely correct. Corrected these.